Description: Use sensible-browser
 Change default browser to /usr/bin/sensible-browser.
Author: Julien BLACHE <jblache@debian.org>
Forwarded: not-needed

Index: xsane-0.998/src/xsane.h
===================================================================
--- xsane-0.998.orig/src/xsane.h	2011-02-04 19:50:22.945016002 +0100
+++ xsane-0.998/src/xsane.h	2011-02-04 19:50:32.653016002 +0100
@@ -251,7 +251,7 @@
 # elif defined(HAVE_OS2_H)
 #  define DEFAULT_BROWSER "netscape"
 # else
-#  define DEFAULT_BROWSER "netscape"
+#  define DEFAULT_BROWSER "/usr/bin/sensible-browser"
 # endif
 #endif
 
